xnat-data-structure-requirements-specification.md

```diff
@@ -4,20 +4,80 @@
 
 ...
 
-**Goal**: give a structured description and insight in how non-raw DICOM and non-DICOM data (e.g., converted DICOM, segmentations, derived data) in a project is structured, what the files contain, and allow users to expose this structure through the API to be used with tooling like [xnatpy](https://xnat.rtfd.io). 
+**Goal**: give a structured description and insight in how non-raw DICOM
+and non-DICOM data (e.g., converted DICOM, segmentations, derived data) 
+in a project is structured, what the files contain, and allow users to
+expose this structure through the API to be used with tooling like 
+[xnatpy](https://xnat.rtfd.io). 
+
+## Motivation
+
+The XNAT medical image storage solution has defined a data structure for
+medical imaging data. This is following the concepts/levels of `project`,
+`subject`, `experiment`, `scan`, and `resource`. This gives the data it's
+general structure. The names of each level and the contents are not
+pre-defined. This allows flexibility for users to store any kind of data in
+the archive and match their use-case, but also makes the data hard to interpret
+without additional information. Getting access to a project on XNAT will not
+make you understand the data further than the general structure.
+
+For data to be FAIR the data should be well described so that new users can
+understand and use the data. This requires a more comprehensive description of
+the data in an archive. For this we propose to create a *data-structure file*
+for each project on XNAT, which describes what data is available for that 
+project.
+
+Such a file will not only allow people to immediately know what data is in a 
+project, but also allow the creation of tools to help with data-management. 
+These tools could include tools to validate data, select and process data, 
+storing data under the right place automatically and more.
+
+## Impact
+
+The choice for a data-structure file to improve the data structure in XNAT is
+inspired by limiting the negative impact on users, while allowing for the 
+advantages via opt-in.
+
+* This proposal will only add optional functionality and not hamper existing
+  users in any way. They will only be affected if they opt in to use the
+  data-structure file.
+* Projects with a data-structure file will have:
+  * their data described in a way that is both human and machine-readable
+  * be able to use any tools implementing this definition
+  * be (more) FAIR
```

Comment:
The formulation could be a bit stronger here in my opinion, something like:
* The data structure file will not be a requirement for projects in XNAT, but we want to strongly encourage people collecting and/or generating data to use this file for Findability and Interoperability of their data.
* Projects with a data-structure file and data structure validation will:
  * have it's data described in a way that is both human and machine-readable
  * be able to benefit from tooling that will make data management more consistent and less error-prone
  * have happy data-managers: managing the data will be more straight-forward and give data structure documentation at their fingertips
  * have happy data users: they know what can expect and spend less time data-munging
  * have Findable data by default